Rock, paper, scissors/script.js

```diff
@@ -1,24 +1,24 @@
-/*
-This is my first JS project — a simple "Rock, paper, scissors" game for DevTools console
-Let's get you through it block by block, bloke
+function main() {
+  let playerSelection = getPlayerSelection()
+  let computerSelection = getComputerSelection();
 
-First, I wrote the function that will be responsible for the computer's choice in every round
-Each time machine randomly chooses rock, paper or scissors in its' eternal struggle against humanity
-*/
+  function playRound(playerSelection, computerSelection) {
+    if (playerSelection === computerSelection) {
+        return `Computer chose ${playerSelection} too. It's a tie!`;
+    } else if (playerSelection === "rock" && computerSelection === "scissors" || 
+               playerSelection === "paper" && computerSelection === "rock" ||
+               playerSelection === "scissors" && computerSelection === "paper") {
```

Comment:
Сложные условия можно выносить в отдельные функции или переменные. Например:
```suggestion
    } else if isPlayerWinner(playerSelection, computerSelection) {
       ...
       
function isPlayerWinner(playerSelection, computerSelection) {
   if  (playerSelection === "rock" && computerSelection === "scissors" || 
     playerSelection === "paper" && computerSelection === "rock" ||
     playerSelection === "scissors" && computerSelection === "paper") {
     return true;
   }
   return false;
}
```

или в переменные:
```suggestion
    isTie = playerSelection === computerSelection
    isPlayerWinner = (playerSelection === "rock" && computerSelection === "scissors" || 
      playerSelection === "paper" && computerSelection === "rock" ||
      playerSelection === "scissors" && computerSelection === "paper")
    if (isTie) {
       return `Computer chose ${playerSelection} too. It's a tie!`;
    } else if (isPlayerWinner) {
    ...
```